commit 1684dd6532512dc067d33de7b0569f75dc408151
Author: Kirill A. Korinsky <kirill@korins.ky>
Date:   Fri Nov 19 15:51:18 2021 +0100

    Prevent fails of tests when path contains dots
    
    Unit test may fails when path to temprorary library contains `.`; to
    garantue that bug wasn't here, it forces to use one more `.` inside path.
    
    Fixes: https://github.com/beetbox/beets/issues/4151

diff --git a/test/test_library.py b/test/test_library.py
index da7d745e..68c1744a 100644
--- a/test/test_library.py
+++ b/test/test_library.py
@@ -224,7 +224,7 @@ class DestinationTest(_common.TestCase):
         self.i.album = '.something'
         dest = self.i.destination()
         self.assertTrue(b'something' in dest)
-        self.assertFalse(b'/.' in dest)
+        self.assertFalse(b'/.something' in dest)
 
     def test_destination_preserves_legitimate_slashes(self):
         self.i.artist = 'one'
